components/create_comment/create_comment.tsx

```diff
@@ -462,7 +457,6 @@ class CreateComment extends React.PureComponent<Props, State> {
     }
 
     handleNotifyAllConfirmation = () => {
-        this.props.closeModal(ModalIdentifiers.NOTIFY_CONFIRM_MODAL);
```

Comment:
This modal is now responsible for closing itself instead of `CreateComment` and `CreatePost` having to do it